__tests__/app.test.js

```diff
@@ -37,13 +38,56 @@ describe('GET /api/topics', () => {
 			.get('/api/topics')
 			.expect(200)
 			.then(({ body }) => {
-				for (item of body) {
-					expect(item).toMatchObject({
+				const { topics } = body;
+				expect(topics.length).toBe(3);
+				for (const topic of topics) {
+					expect(topic).toMatchObject({
 						slug: expect.any(String),
 						description: expect.any(String),
 					});
 				}
-				expect(body.length).toBe(3);
+			});
+	});
+});
+
+describe('GET /api/articles/:article_id', () => {
+	test('200: return an article when a valid id is supplied', () => {
+		const article_id = 3;
+		return request(app)
+			.get(`/api/articles/${article_id}`)
+			.expect(200)
+			.then(({ body }) => {
+				const { articles } = body;
+				for (const article of articles) {
+					expect(article).toMatchObject({
+						article_id: expect.any(Number),
```

Comment:
Do we expect any number for the article_id when we request a specific article? We could assert the actual number here to make sure we get the correct article.